src/Tribe/Events/Views/V2/Models/Tickets.php

```diff
@@ -189,7 +189,7 @@ public function fetch_data() {
 							$text = _n( '%1$s spot left', '%1$s spots left', $stock, 'event-tickets' );
 						} else {
 							/* translators: %1$s: Number of stock, %2$s: Ticket label, %3$s: Tickets label */
-							$text = _n( "%1$s %2$s left", "%1$s %3$s left", $stock, 'event-tickets' ); // phpcs:ignore
+							$text = _n( '%1$s %2$s left', '%1$s %3$s left', $stock, 'event-tickets' );
```

Comment:
:warning: Mismatched placeholders is probably an error

<sub>hash: 8a757c1133745b7f32a5854c84259b37</sub>